packages/editor/src/client/api/user.graphql

```diff
@@ -1,106 +1,125 @@
 fragment FullUserSubscription on UserSubscription {
-    memberPlan {
```

Comment:
Are these changes from prettier or your code editor?

Reply:
@roosnic1  Code editor, is that correct?

Reply:
@roosnic1 code editor but i changed that to just have my addition. I am not sure what's the right? 
If it's needed, it should have been done by husky right?